contrib/gitian/README.md

```diff
@@ -233,14 +229,15 @@ This will create a `.sig` file for each `.assert` file above (2 files for each p
 Submitting your signed assert files
 -----------------------------------
 
-Make a pull request (both the `.assert` and `.assert.sig` files) to the
-[monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository:
+Fork the [monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository and clone the fork. Add both the `.assert` and `.assert.sig` files, as well as your public GPG key (`.asc`) to `gitian-pubkeys` directory. After pushing, create a PR to merge to the upstream.
 
 ```bash
+git clone https://github.com/$GH_USER/gitian.sigs.git
+cd gitian.sigs
 git checkout -b $VERSION
-# add your assert and sig files...
+# add your assert and sig files, as well as public PGP key...
 git commit -S -a -m "Add $GH_USER $VERSION"
-git push --set-upstream $GH_USER $VERSION
+git push --set-upstream origin $VERSION
```

Comment:
Fwiw, I usually clone monero-project/foo as the origin, and my fork'd repo is named "mine". It's handier to have monero-project as the origin, because then you can do a single `git pull` later to bring yourself up to date. And then when submitting your files just `git push mine $VERSION` which will give you a link to create the pull request.

Reply:
So the initial steps should be: fork gitian.sigs repo on github. Then:
````
git clone https://github.com/monero-project/gitian.sigs/
cd gitian.sigs
git remote add $GH_USER https://github.com/$GH_USER/gitian.sigs
````
Followed by the original text.